source/common/json/constants.h

```diff
@@ -11,6 +11,15 @@ class Constants {
   static constexpr absl::string_view True = "true";
   static constexpr absl::string_view False = "false";
   static constexpr absl::string_view Null = "null";
+
+  // Constants for common JSON delimiters.
+  static constexpr absl::string_view DoubleQuote = "\"";
+  static constexpr absl::string_view Comma = ",";
+  static constexpr absl::string_view Colon = ":";
+  static constexpr absl::string_view ArrayBegin = "[";
+  static constexpr absl::string_view ArrayEnd = "]";
+  static constexpr absl::string_view MapBegin = "{";
+  static constexpr absl::string_view MapEnd = "}";
```

Comment:
I initially think I can avoid this by re-using the `Level` api of `Streamer`. But when I try to construct `FormatElements`. I found the low level control to the deilimiter is still necessary.